docs/language.rst

```diff
@@ -643,6 +643,33 @@ To prevent string interpolation, use raw strings
     motd = r"Welcome to {{hostname}}\n"
 
 
+String formatting
```

Comment:
I recall from the demo meeting that our f-string support had some limitations with respect to the f-string support in Python. Is that true? If yes, we need to mention these limitations here.

Reply:
Added the limitation regarding `f"{arg=}"`